src/NexusMods.App.UI/Pages/LoadoutPage/LoadoutViewModel.cs

```diff
@@ -112,32 +162,34 @@ public class LoadoutTreeDataGridAdapter : TreeDataGridAdapter<LoadoutItemModel,
     private readonly Dictionary<LoadoutItemModel, IDisposable> _commandDisposables = new();
 
     private readonly IDisposable _activationDisposable;
+
     public LoadoutTreeDataGridAdapter(IServiceProvider serviceProvider)
     {
         _loadoutDataProviders = serviceProvider.GetServices<ILoadoutDataProvider>().ToArray();
         _connection = serviceProvider.GetRequiredService<IConnection>();
 
         _activationDisposable = this.WhenActivated(static (adapter, disposables) =>
-        {
-            Disposable.Create(adapter._commandDisposables, static commandDisposables =>
             {
-                foreach (var kv in commandDisposables)
-                {
-                    var (_, disposable) = kv;
-                    disposable.Dispose();
-                }
+                Disposable.Create(adapter._commandDisposables,
+                        static commandDisposables =>
```

Comment:
What the hell happened with the indentation.

Reply:
The auto formatter must have kicked in when I moved the function.
Yeah, it wants to split it on two lines for some reason.